packages/legacy-client/src/ontology-runtime/ontologyProvider/Errors.ts

```diff
@@ -58,12 +58,9 @@ export type GetObjectError =
   | ObjectNotFound
   | PropertiesNotFound
   | CommonApiError;
-export type ListObjectsError =
-  | ObjectsExceededLimit
-  | ObjectTypeNotFound
-  | ObjectTypeNotSynced
-  | OntologySyncing
-  | CommonApiError;
+
+export type ListObjectsError = LoadObjectSetError;
```

Comment:
This should be a non-breaking change to only have `LoadObjectSetError` since we do all those through one endpoint